keyboards/prototypist/pt60/info.json

```diff
@@ -0,0 +1,237 @@
+{
+    "manufacturer": "Proto[Typist]",
+    "keyboard_name": "PT-60",
+    "maintainer": "Anjheos",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B12", "B14", "A8", "A9", "A15", "B3", "B7", "B4", "B5", "B6", "A4", "A5", "A6", "A7" ],
+        "rows": ["B0", "B1", "B2", "B10", "B11"]
+    },
+    "processor": "STM32F303",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x4A46"
+    },
```

Comment:
```suggestion
    },
    "community_layouts": ["60_iso", "LAYOUT_60_ansi"],
```

Reply:
Thanks for clarifying that! I was just about to ask if you meant "60_iso" or "LAYOUT_60_iso"